src/assets/config/appconfig.json

```diff
@@ -22,6 +22,11 @@
     "feedbackEmailId": "products.feedback@UKHO.gov.uk",
     "apiUrl": "",
     "stateManagementApiUrl": "",
-    "batchAttributes": []
+    "batchAttributes": [
```

Comment:
I would like to see some sample config here (perhaps add the same config as used in the show and tell), but if it causes extra work then please leave it blank (I wouldn't be surprised if the functional tests failed, if so leave it blank)